test/micromamba/test_config.py

```diff
@@ -500,25 +500,55 @@ def test_file_append_single_input(self):
         ).splitlines() == ["channels:", "  - flowers"]
 
     def test_file_append_multiple_inputs(self):
+        with open(TestConfigModifiers.home_rc_path, "w") as f:
+            f.write("channels:\n  - foo")
+
+        config(
+            "append",
+            "channels",
+            "condesc,mambesc",
```

Comment:
Is there a difference between adding args as an independent string and as you did?
Is this just to remove the `splitlines()` command?

Reply:
sorry I don't get it, could you please point me what modification you're focusing on ?

Reply:
Just wondering because before it was

```
config("append", "channels", "a", "b")
```
but now you changed to:

```
config("append", "channels", "a,b")
```
and I'm asking why.

Reply:
The previous behavior was to pick the last value only, and drop the intermediate one (`a` in that case).
I changed the behavior to:
- be able to handle multiple key/val pairs (as vector of string pairs)
- use the exact same `deserialization` used for environment variable. So there is no surprise you can use the same syntax when you set `CONDA_CHANNELS` to `a,b`